### PR TITLE
NodeJS Course - Conclusion lesson: Change broken link

### LINKED ISSUE #28388: Conclusion: Broken link
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

The link to Droplets under the DevOps section is broken [Droplets](https://docs.digitalocean.com/products/droplets/quickstart) The correct link is now [Correct link](https://docs.digitalocean.com/products/droplets/getting-started/quickstart/)

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/nodejs-conclusion

### (Optional) Discord Name

gokhan.reads

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by Gohan61**
## Because
Previous link to Droplets by Digital Ocean was broken and not leading to correct page.

## This PR
• Changes Droplets link under the DevOps section

## Issue
Closes #28388

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: react/introduction/setting_up_a_react_environment.md [hunk 1]
```diff
@@ -154,5 +154,4 @@ The following questions are an opportunity to reflect on key topics in this less
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- [Vite React App with EsLint & Prettier in VSCode (2023)](https://www.youtube.com/watch?v=SMbqi1HPprc) is a helpful video guide on how to set up Prettier inside your Vite-React projects.
```
Comment (JoshDevHub) [inline react/introduction/setting_up_a_react_environment.md:157]: Why has this content been deleted? Maybe related to a different contribution you've made or something? Seems completely unrelated to your link change in the conclusion lesson

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Thanks for fixing the Digital Ocean link :raised_hands:

Just one comment about an extra change you have in the "Setting Up a React Environment" lesson.

File: react/introduction/setting_up_a_react_environment.md (hunk 1 above)
Comment (Gohan61) [reply react/introduction/setting_up_a_react_environment.md:157]: This was part of another fix that was merged a while ago [Issue #27878](https://github.com/TheOdinProject/curriculum/pull/27878). I fetched from upstream on my local clone but did not merge hence that commit, my bad.

**Review by JoshDevHub — APPROVED** [review]
